official/recommendation/ncf_main.py

```diff
@@ -403,7 +403,7 @@ def define_ncf_flags():
           "achieved by MLPerf implementation."))
 
   flags.DEFINE_bool(
-      name="ml_perf", default=None,
```

Comment:
When running the model, it spawns a subprocess by running `python data_async_generation.py <arguments>`. We pass the ml_perf flag to the subprocess. When the default was None, we passed `--ml_perf=None`, which is not allowed. You're only allowed to pass `--ml_perf=True` or `--ml_perf=False` to the subprocess, since ml_perf is a boolean argument. So changing the default to `False` will pass `--ml_perf=False`, fixing the issue.